GDJS/Runtime/pixi-renderers/layer-pixi-renderer.ts

```diff
@@ -72,10 +72,26 @@ namespace gdjs {
       const centerY =
         this._layer.getCameraX() * zoomFactor * sinValue +
         this._layer.getCameraY() * zoomFactor * cosValue;
-      this._pixiContainer.position.x = -centerX;
-      this._pixiContainer.position.y = -centerY;
-      this._pixiContainer.position.x += this._layer.getWidth() / 2;
-      this._pixiContainer.position.y += this._layer.getHeight() / 2;
+      this._pixiContainer.position.x = this._layer.getWidth() / 2 - centerX;
+      this._pixiContainer.position.y = this._layer.getHeight() / 2 - centerY;
+
+      if (
+        this._layer.getRuntimeScene().getGame().getPixelsRounding() &&
+        (cosValue === 0 || sinValue === 0) &&
+        Number.isInteger(zoomFactor)
+      ) {
+        // Camera rounding is important for pixel perfect games.
+        // Otherwise the camera position fractional part is added to
+        // the sprite one and it changes in which direction sprites are rounded.
+        // It makes sprites rounding inconsistent with each other
+        // and they seems to move on pixel left and right.
+        this._pixiContainer.position.x = Math.round(
+          this._pixiContainer.position.x
+        );
+        this._pixiContainer.position.y = Math.round(
+          this._pixiContainer.position.y
+        );
```

Comment:
Actually, PIXI uses `floor`. We will have to use floor too. This will avoid flickering when centering the camera on an object in the most common cases: when the frame and the sprite dimension parity is the same.

We could then suggest game creators to use even dimension for screen and sprites (or integer center) when using pixel rounding.

Reply:
So the conclusion is that we should use floor here and make a note/comment explaining that this is to use the same rounding as PixiJS (can you explain at which level/for which primitive this rounding is used in PIxiJS)?
Thanks!